README.md

```diff
@@ -367,7 +367,7 @@ We are looking for maintainers to add more parsers and to write query files for
 - [ ] [scheme](https://github.com/6cdh/tree-sitter-scheme)
 - [x] [scss](https://github.com/serenadeai/tree-sitter-scss) (maintained by @elianiva)
 - [x] [slang](https://github.com/theHamsta/tree-sitter-slang) (experimental, maintained by @theHamsta)
-- [x] [slint](https://github.com/jrmoulton/tree-sitter-slint) (experimental, maintained by @jrmoulton)
+- [x] [slint](https://github.com/slint-ui/slint) (experimental, maintained by @hunger)
```

Comment:
Do you still want to keep it "experimental"? If you're going to keep maintaining it, you can (and should) remove that.